test/proto_rules/go_grpc_ff/proto/service.proto

```diff
@@ -1,5 +1,7 @@
 syntax = "proto3";
 
+option go_package = "github.com/thought-machine/please/test/proto_fules/go_frpc_ff/proto/service";
```

Comment:
This actually ends up as `github.com/thought-machine/please/test/proto_fules/go_frpc_ff/proto` because that's what the name of the grpc_libarary() is.